website/coronal_holes/index.php

```diff
@@ -0,0 +1,6 @@
+<?
+
+	header("Location: https://solarmonitor.org/chimera.php"); /* Redirect browser */
+	exit();
+
+?>
```

Comment:
I don't really like this types of redirects. This creates many files that then become harder and harder to maintain. Maybe a `.htaccess` file is more efficient or some rules in the apache/nginx config files there you would get a single file with all the redirects you want to do, so in case you need to change it is only one file to change, not files across multiple directories.